addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -391,11 +386,11 @@ test("Current user that is a follower should be considered as such", async () =>
     await contains(".o-mail-Composer-suggestion", { count: 5 }); // FIXME: should be 3, but +2 extra with Hermit / Public User
     await contains(".o-mail-Composer-suggestion", {
         text: "Mitchell Admin",
-        before: [".o-mail-Composer-suggestion", { text: "Person B(b@test.com)" }],
+        before: [".o-mail-Composer-suggestion", { text: "Person B" }],
     });
     await contains(".o-mail-Composer-suggestion", {
-        text: "Person B(b@test.com)",
-        before: [".o-mail-Composer-suggestion", { text: "Person A(a@test.com)" }],
+        text: "Person B",
+        before: [".o-mail-Composer-suggestion", { text: "Person A" }],
     });
```

Comment:
Why removing coverage of email in the partner suggestion?

Reply:
we removed the `()` so there's a diff anyway, and i think `Person A` looks better than `Person Aa@test.com` but i can revert it anyway.

Reply:
I don't think the email is that useful, all suggested persons have a name anyway and replacing the email by the avatar is better.

LGTM then :)